src/index.js

```diff
@@ -21,14 +21,15 @@ import './index.css';
 import store from "./store/index";
 import { Provider } from "react-redux";
 
-ReactDOM.render(
+const container = document.getElementById('root');
+const root = createRoot(container);
+root.render(
```

Comment:
This is the React 18 syntax of building the root element. ( [Docs](https://react.dev/blog/2022/03/08/react-18-upgrade-guide#updates-to-client-rendering-apis) )